src/main/resources/db/changelog/changes/08-add-default-users-if-not-exist.yaml

```diff
@@ -0,0 +1,62 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-admin-if-not-exist
+      author: mpbakunov
+      preConditions:
+        - onFail: CONTINUE
+        - sqlCheck:
+            expectedResult: 0
+            sql: "SELECT COUNT(*) FROM users_roles INNER JOIN roles ON users_roles.role_id = roles.id
+             WHERE roles.name = 'ADMIN'"
+      changes:
```

Comment:
Why do you need this script? You have already added a user with the admin role in the previous script

Reply:
**This script was necessary in the case when there were users in the User table before the new changes were set.** The goal is to determine whether there is a user with ADMIN roles and add default admin user with ADMIN role if it is absent.